website/docs/reference/warehouse-profiles/mssql-profile.md

```diff
@@ -28,17 +28,44 @@ pip install dbt-sqlserver
 
 ### Prerequisites
 
-On Ubuntu make sure you have the ODBC header files before installing
+On Debian/Ubuntu make sure you have the ODBC header files before installing
 
-    sudo apt install unixodbc-dev
+```bash
+sudo apt install unixodbc-dev
+```
+
+Download and install the [Microsoft ODBC Driver 18 for SQL Server](https://docs.microsoft.com/en-us/sql/connect/odbc/download-odbc-driver-for-sql-server?view=sql-server-ver15)
+
+The adapter is tested with SQL Server 2017, SQL Server 2019, SQL Server 2022 and Azure SQL Database. These versions are tested with Microsoft ODBC Driver 17 and Microsoft ODBC Driver 18.
+
+## Authentication methods & profile configuration
+
+### Common configuration
+
+For all the authentication methods below, the following configuration options can be set in your `profiles.yml` file:
 
-Download and install the [Microsoft ODBC Driver 17 for SQL Server](https://docs.microsoft.com/en-us/sql/connect/odbc/download-odbc-driver-for-sql-server?view=sql-server-ver15)
+* `driver`: The ODBC driver to use. E.g. `ODBC Driver 18 for SQL Server`
+* `server`: The server hostname. E.g. `localhost`
+* `port`: The server port. E.g. `1433`
+* `database`: The database name.
+* `schema`: The schema name. E.g. `dbo`
+* `retries`: The number of automatic times to retry a query before failing. Defaults to `1`. Note that queries with syntax errors will not be retried. This setting can be used to overcome intermittent network issues.
+* `encrypt`: Whether to encrypt the connection to the server. Defaults to `true`. Read more about encryption [below](#connection-encryption).
+* `trust_cert`: Whether to trust the server certificate. Defaults to `false`. Read more about encryption [below](#connection-encryption).
 
-### Authentication methods
+### Connection encryption
 
-#### Standard SQL Server authentication
+Microsoft made several changes in the release of ODBC Driver 18 that affects how connection encryption is configured. To accomodate these changes, dbt-sqlserver 1.2.0 or newer also changed the defaults for the `encrypt` and `trust_cert` configuration options. Both options are now always included in the connection string to the database.
 
-SQL Server credentials are supported for on-prem as well as Azure,
+The default value of `encrypt` is `true`, meaning that connections are encrypted by default.
+
+The default value of `trust_cert` is `false`, meaning that the server certificate will be validated. By setting this to `true`, a self-signed certificate will be accepted.
+
+More details about how these values affect your connection and how they are used differently in versions of the ODBC driver can be found in the [Microsoft documentation](https://learn.microsoft.com/en-us/sql/connect/odbc/dsn-connection-string-attribute?view=sql-server-ver16#encrypt).
```

Comment:
ok now i see the encryption piece is a whole section. maybe make the callout tip piece at the top of the page and link to this section instead?